### ISSUE TITLE
JsonParse error missing field `inh`

### ISSUE THREAD

**Issue by glenjamin** [sbdchd/squawk#79]
When running on the following file
```
ALTER TABLE ONLY public.tasks
    DROP CONSTRAINT tasks_fk,
    ADD CONSTRAINT tasks_fk
        FOREIGN KEY (job_id) REFERENCES public.jobs(external_id)
            ON DELETE CASCADE NOT VALID;

ALTER TABLE public.tasks VALIDATE CONSTRAINT tasks_fk;
```

I get the following error:
```
Err(
    CheckSQL(
        ParsingSQL(
            JsonParse(
                "missing field `inh` at line 1 column 150",
            ),
        ),
    ),
)
```

In addition, when passing multiple files at once, the parse error doesn't include a hint about which file was wrong (I figured this out using strace).

Any ideas what's up?

**Comment by sbdchd** [comment]
Thanks for the report! I'm looking into this

Squawk currently parses the json from the libpg_query library, but there is a mismatch with the schema.

Typically I use the `--dump-ast=raw` and `--dump-ast=parsed` to debug the issue:

```json
❯ squawk --dump-ast=raw foo.sql | jq .
[
  {
    "RawStmt": {
      "stmt": {
        "AlterTableStmt": {
          "cmds": [
            {
              "AlterTableCmd": {
                "behavior": 0,
                "name": "tasks_fk",
                "subtype": 22
              }
            },
            {
              "AlterTableCmd": {
                "behavior": 0,
                "def": {
                  "Constraint": {
                    "conname": "tasks_fk",
                    "contype": 8,
                    "fk_attrs": [
                      {
                        "String": {
                          "str": "job_id"
                        }
                      }
                    ],
                    "fk_del_action": "c",
                    "fk_matchtype": "s",
                    "fk_upd_action": "a",
                    "location": 68,
                    "pk_attrs": [
                      {
                        "String": {
                          "str": "external_id"
                        }
                      }
                    ],
                    "pktable": {
                      "RangeVar": {
                        "inh": true,
                        "location": 128,
                        "relname": "jobs",
                        "relpersistence": "p",
                        "schemaname": "public"
                      }
                    },
                    "skip_validation": true
                  }
                },
                "subtype": 14
              }
            }
          ],
          "relation": {
            "RangeVar": {
              "location": 17,
              "relname": "tasks",
              "relpersistence": "p",
              "schemaname": "public"
            }
          },
          "relkind": 37
        }
      },
      "stmt_len": 192
    }
  },
  {
    "RawStmt": {
      "stmt": {
        "AlterTableStmt": {
          "cmds": [
            {
              "AlterTableCmd": {
                "behavior": 0,
                "name": "tasks_fk",
                "subtype": 18
              }
            }
          ],
          "relation": {
            "RangeVar": {
              "inh": true,
              "location": 207,
              "relname": "tasks",
              "relpersistence": "p",
              "schemaname": "public"
            }
          },
          "relkind": 37
        }
      },
      "stmt_len": 55,
      "stmt_location": 193
    }
  }
]
```

```
❯ squawk --dump-ast=parsed foo.sql | jq .
PGQuery(
    JsonParse(
        "missing field `inh` at line 1 column 150",
    ),
)
```

**Comment by sbdchd** [comment]
Fix should be up on NPM now via version 0.5.4
